app/services/doc_auth/response.rb

```diff
@@ -103,5 +103,10 @@ def doc_auth_success?
       # to be implemented by concrete subclass
       false
     end
+
+    def selfie_status
```

Comment:
Same comment as in `application_helper.rb` above: Can we do this in a different PR?